README.md

```diff
@@ -176,7 +176,7 @@ Heimdall's frontend container image is distributed on [DockerHub](https://hub.do
 5. Run the following commands in a terminal window from the Heimdall source directory. For more information on the .env file, visit [Environment Variables Configuration.](https://github.com/mitre/heimdall2/wiki/Environment-Variables-Configuration)
    ```bash
    ./setup-docker-env.sh
```

Comment:
We should add a note about windows vs. linux/mac because there is a different setup script for windows.

Reply:
I created a windows setup docker batch file. Can't attached it here because the extension is not supported for this widget, will attach it to the slack posting.